pkg/collect/host_run.go

```diff
@@ -54,26 +101,20 @@ func (c *CollectHostRun) Collect(progressChan chan<- interface{}) (map[string][]
 		}
 	}
 
-	collectorName := c.hostCollector.CollectorName
-	if collectorName == "" {
-		collectorName = "run-host"
-	}
-	resultInfo := filepath.Join("host-collectors/run-host", collectorName+"-info.json")
-	result := filepath.Join("host-collectors/run-host", collectorName+".txt")
+	resultInfo := filepath.Join(collectorRelativePath, "collector-info.json")
```

Comment:
You probably want to revert this file names to what they were.

Reply:
makes sense!